providers/writing_a_new_provider.md

```diff
@@ -339,9 +342,7 @@ class ManageIQ::Providers::AwesomeCloud::CloudManager < ManageIQ::Providers::Clo
     ManageIQ::Providers::AwesomeCloud::Regions.all.map { |region| {:label => region[:name], :value => region[:name]} }
   end
 
-  supports :regions
   validates :provider_region, :inclusion => {:in => ManageIQ::Providers::AwesomeCloud::Regions.names}
-end
```

Comment:
This isn't actually the entire class, there are a number of other methods built by the provider generator so removing the `end` here to not make it seem like this is all you need